stable/k8s-secret-updater/templates/app-deploy.yaml

```diff
@@ -26,7 +26,12 @@ spec:
       annotations:
         checksum/secret: {{ include (print $.Template.BasePath "/app-secret.yaml") . | sha256sum }}
     spec:
+    {{ if .Values.serviceAccount.name }}
```

Comment:
Conditionals should not create new lines.

Use `{{-` with `if`, `else` and `end`.